packages/web-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,122 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+} from '../theme/design-tokens.js';
+
+/** Radio styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-grid')}
+
+  :host {
+    grid-auto-flow: column;
+    grid-template-columns: max-content;
+    gap: ${spacingHorizontalXS};
+    align-items: center;
+    height: 32px;
+    cursor: pointer;
+    outline: none;
+    position: relative;
+    user-select: none;
+    color: var(--state-color);
+  }
+  .label {
+    cursor: pointer;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+  }
+  .label__hidden {
+    display: none;
+  }
+  .control {
+    box-sizing: border-box;
+    align-items: center;
+    border: 1px solid var(--control-border-color);
+    border-radius: ${borderRadiusCircular};
+    display: flex;
+    height: 16px;
+    justify-content: center;
+    margin: ${spacingVerticalS} ${spacingHorizontalS};
+    position: relative;
+    width: 16px;
+    justify-self: center;
+  }
+  .checked-indicator {
+    border-radius: ${borderRadiusCircular};
+    height: 10px;
+    opacity: 0;
+    width: 10px;
+  }
+  :host([aria-checked='false']:hover) .control {
+    color: ${colorNeutralForeground2};
+  }
+  :host(:focus-visible)::after {
+    border: 2px solid ${colorStrokeFocus1};
+    border-radius: ${borderRadiusMedium};
+    box-shadow: inset 0 0 0 2px ${colorStrokeFocus2};
+    content: '';
+    cursor: pointer;
+    inset: 0px;
```

Comment:
NIT `inset: 0`

Reply:
I removed these styles to be more inline with how the other components are setting the focus state